tensorflow_quantum/core/ops/cirq_ops_test.py

```diff
@@ -423,6 +421,45 @@ def test_sample_empty_circuit(self):
             cirq.Circuit()).SerializeToString()
         _ = test_op([test_empty_circuit], [], [[]], [10])
 
+    def test_get_cirq_samples_general(self):
+        """Tests that a general cirq.Sampler is compatible with sampling."""
+
+        class DummySampler(cirq.Sampler):
+            """Mock general cirq.Sampler."""
+
+            def run_sweep(self, program, params, repetitions):
+                """Returns all ones in the correct sample shape."""
+                return [
+                    cirq.TrialResult.from_single_parameter_set(
+                        params=params,
+                        measurements={
+                            'tfq':
+                                np.array([[1] * len(program.all_qubits())] *
+                                         repetitions,
+                                         dtype=np.bool),
```

Comment:
Could you check it with `dtype=np.int32`, not with `dtype=np.bool` as mentioned above?